### ISSUE TITLE
Request: Telegram icon

### ISSUE THREAD

**Issue by z3ntu** [ionic-team/ionicons#338]

**Comment by bensperry** [comment]
Telegram isn't well known enough to warrant its addition. Putting this on hold for now.

**Comment by z3ntu** [comment]
According to their own numbers they hit about 62 Million users in May last year, and alone during the brazil-whatsapp-block they got 5.7 million new users.

**Comment by bensperry** [comment]
Yeah, I just need to be careful about adding too many icons. With 3.0 we're trying to reduce a lot of the extra icons that weren't used as much in order to reduce file size (which impact load times). Eventually, we might be able to only load in the icons being used in an app, at which point we could add more icons to the project. I'll keep this on the back burner for that.

**Comment by waglds** [comment]
Telegram now has increased the number of users. That would be great having its icon as well. I'm also in need of it!

**Comment by lanabhat** [comment]
I have my app access my telegram group, I think it's a good addition to add a Telegram icon...

**Comment by DonTermi** [comment]
Telegram has over 100 million users since february 2016. I´m voting for adding telegram.

**Comment by alesmit** [comment]
Upvote for telegram icon to be included in Ionicons

**Comment by abrshdir** [comment]
Need Telegram icon!

**Comment by TiagoDanin** [comment]
+1, Telegram logo/icon: https://telegram.org/img/t_logo.png
`ion-paper-airplane` is similar

**Comment by tyl3r** [comment]
Requested in 2016..., 3 years and still no icon for Telegram, is not that hard :|.

**Comment by eladcandroid** [comment]
Any news?

**Comment by aminkhadivar** [comment]
Need Telegram icon!

**Comment by admin-peppersalt** [comment]
Need Telegram

**Comment by allexiusw** [comment]
I agree that Telegram icon is needed.

**Comment by mimali71** [comment]
What is the status on the telegram icon?

**Comment by teito-dev** [comment]
the app has more than half a billion users now , would be great if it has its own spot

**Comment by calvintam236** [comment]
I would love to see this get added soon!

**Comment by calvintam236** [comment]
Official logo can be downloaded from here: https://telegram.org/tour/screenshots
If I have time later, I may submit a PR for this.

**Comment by RiyanRIS** [comment]
need telegram icon.!

**Comment by ebukun** [comment]
Not sure we ever going to get a telegram icon

**Comment by agcentory240** [comment]
need telegram icon

**Comment by hhsm95** [comment]
Maybe 2023 will be the year for Telegram Icon?

**Comment by akdasa** [comment]
Hello guys. It took you seven years, and there is no progress. It's a shame :)

**Comment by TradeExe** [comment]
still waiting for the icon to be added

**Comment by gohkokhan** [comment]
@bensperry  Given how popular Telegram has become now in 2023, I think a lot of developers and users out there would really appreciate having a Telegram icon in the set.

**Comment by akdasa** [comment]
You would have already created the icon while you were closing more and more requests. You would spend the same amount of time.

**Comment by amochkin** [comment]
I don't understand why Xing is being included with their 20M users, while there is a problem adding Telegram with their 800M users, having Telegram icon for Ionic already made: https://svgshare.com/i/ogR.svg
I guess it is solely maintainers subjective decision.

**Comment by christian-bromann** [comment]
Consolidating into #1449
